src/ui/components/shared/Nags/Nags.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import hooks from "ui/hooks";
+import { Nag } from "ui/hooks/users";
+import MaterialIcon from "../MaterialIcon";
+
+function NagHat({
+  mainText,
+  subText,
+  nagType,
+}: {
+  mainText: string;
+  subText: string;
+  nagType: Nag;
+}) {
+  const { nags } = hooks.useGetUserInfo();
+  const isAuthor = hooks.useUserIsAuthor();
+
+  if (nags.includes(nagType) || !isAuthor) {
+    return null;
+  }
+
+  return (
+    <div className="bg-secondaryAccent text-white py-1 px-2 flex space-x-2 items-center leading-tight">
+      <MaterialIcon iconSize="xl">auto_awesome</MaterialIcon>
+      <div className="text-xs flex flex-col">
+        <div>{mainText}</div>
+        <div className="font-bold">{subText}</div>
+      </div>
+    </div>
+  );
+}
+
+export function EditorNag() {
+  const { nags } = hooks.useGetUserInfo();
+
+  // Don't show the editor nag until the user has gotten past the
+  // console navigate nag.
+  if (!nags.includes(Nag.FIRST_CONSOLE_NAVIGATE)) {
```

Comment:
@jazzdan @ryanjduffy is it easy/possible to set up a launch darkly A/B test for this.

I'd love to say half of our users should see this NAG regardless of whether they've first clicked on console navigate.

Reply:
should be. I added support for LD in devtools before launch so it should be a bit of work on the LD side and a bit in code but nothing too bad

Reply:
Great!

https://github.com/RecordReplay/devtools/blob/a0c76b5d2ee4265f8eb7c3f87d106770cadca546/src/ui/components/shared/WorkspaceSettingsModal/BillingBanners.tsx#L3-L4